fluss-client/src/main/java/org/apache/fluss/client/metadata/MetadataUpdater.java

```diff
@@ -316,14 +335,61 @@ private static Cluster initializeCluster(Configuration conf, RpcClient rpcClient
         return cluster;
```

Comment:
We should improve this exception message, it didn't carry exception message to the top error message which may confuse users. 

```
            String errorMsg =
                    "Failed to initialize fluss client connection to bootstrap servers: "
                            + inetSocketAddresses
                            + ". \nReason: "
                            + lastException.getMessage();
```